dietpi/patch_file

```diff
@@ -1551,6 +1517,17 @@ Pin-Priority: -1' > /etc/apt/preferences.d/dietpi-docker_fix
 			[[ -f /etc/apache2/sites-available/dietpi-owncloud.conf ]] && a2ensite dietpi-owncloud
 			[[ -f /etc/apache2/sites-available/dietpi-nextcloud.conf ]] && a2ensite dietpi-nextcloud
 			#-------------------------------------------------------------------------------
+			#Clear install state for Medusa
+			(( $G_DIETPI_VERSION_RC == 0 && $G_DIETPI_INSTALL_STAGE == 2 )) && G_CONFIG_INJECT 'aSOFTWARE_INSTALL_STATE\[116\]=' 'aSOFTWARE_INSTALL_STATE[116]=0' /DietPi/dietpi/.installed
```

Comment:
@MichaIng 

Don't believe we can use ```G_DIETPI_VERSION_RC ``` here? EG:

- v6.19.7
- Update to v6.20.2
- ```G_DIETPI_VERSION_RC ``` is taken from server version
- ```G_DIETPI_VERSION_RC == 0``` never triggered.

Might be easier to simply?
```
if [[ ! -d $G_FP_DIETPI_USERDATA/medusa ]]; then

 G_CONFIG_INJECT 'aSOFTWARE_INSTALL_STATE\[116\]='  'aSOFTWARE_INSTALL_STATE[116]=0' /DietPi/dietpi/.installed

fi
```

Reply:
@Fourdee 
Thought to add this for beta testers. If they test Medusa, it's install state should not be reverted on every update, so just do this once on initial beta?

But yeah you are right, RC == 0 is only the case, AFTER first beta release (if we leave RC version == 0).

Check `/DietPi/dietpi/.version` for line 3 `G_DIETPI_VERSION_SUB<=19`?